index.html

```diff
@@ -285,7 +285,7 @@ <h4>Tools</h4>
 				 <p>Not sure what's your next step? Try our <a href='https://w3c.github.io/transitions/nextstep.html'>step finder</a> and look at <a href="https://w3c.github.io/spec-releases/milestones/">milestones calculator</a>.</p>
       </form>
 
-<h2 id="sotd">About This Document</h2>
+gid="sotd">About This Document</h2>
```

Comment:
undo please. I don't hink you meant to change that line.
```suggestion
<h2 id="sotd">About This Document</h2>
```